src/tests/ftest/container/snapshot.py

```diff
@@ -421,110 +424,105 @@ def test_snapshots(self):
                 self.fail(
                     "##Error when retrieving the snapshot data: %s"
                     , str(error))
-            self.log.info("==container_epoch= %s", epoch)
-            self.log.info("==snapshot_list[ind]=%s"
+            self.log.info("  ==container_epoch= %s", epoch)
+            self.log.info("  ==snapshot_list[ind]=%s"
                           , snapshot)
-            self.log.info("==snapshot.epoch=  %s"
+            self.log.info("  ==snapshot.epoch=  %s"
                           , snapshot.epoch)
-            self.log.info("==written thedata size= %s"
+            self.log.info("  ==written thedata size= %s"
                           , len(thedata)+1)
-            self.log.info("==written thedata=%s", thedata)
-            self.log.info("==thedata3.value= %s", thedata3.value)
+            self.log.info("  ==written thedata=%s", thedata)
+            self.log.info("  ==thedata3.value= %s", thedata3.value)
```

Comment:
You may want consider, given the repetition of this code (here, 474-479, 513-518), defining a class method to display this data - you could make some inputs optional to handle the few different output messages.

Reply:
Plese see the new script commit for the fixes.